src/SSHDebugPS/Docker/DockerConnection.cs

```diff
@@ -12,15 +12,85 @@ namespace Microsoft.SSHDebugPS.Docker
 {
     internal class DockerConnection : PipeConnection
     {
+        #region Statics
+
+        internal const string SshPrefix = "ssh=";
```

Comment:
These strings are user visible? If so, I would suggest being a bit more forgiving about parsing them - we should probably allow whitespace after all the tokens which gets removed.

Reply:
so trim before and after the '='? I can do that.